application/controllers/admin/SurveymenuEntryController.php

```diff
@@ -404,6 +411,7 @@ public function reorder()
      * @param integer $id the ID of the model to be loaded
      * @return SurveymenuEntries the loaded model
      * @throws CHttpException
+     * * @deprecated do not use this function in future
```

Comment:
:+1: sorry …

Reply:
@c-schmitz gave me already a go to merge it. Thanks a lot for your review Denis :). This fix is also already merged into 5.x.